packages/core/test/financial-templates/perpetual-multiparty/PerpetualPositionManager.js

```diff
@@ -815,14 +815,29 @@ contract("PerpetualPositionManager", function(accounts) {
     await setFundingRateAndAdvanceTime("0");
     await positionManager.applyFundingRate();
 
-    // Clock has been advanced during the proposal by 10k seconds. The fee rate is one one-thousandth of a percent.
+    // Clock has been advanced during the proposal by 10k seconds. The reward rate is one one-thousandth of a percent.
```

Comment:
This test was a bit unclear, its title is "Basic funding rate fees", which reference the fact that sponsors' locked collateral is taxed to pay funding rate rewards.